### PR TITLE
XCloud local import/export

### PR DESCRIPTION
**Pull request by ChuckL**
XCloud can now do local export and import.
This will bypass the 5MB limit of uploading to XCloud.

### CODE REVIEW

File: Extensions/xcloud.js [hunk 1]
```diff
@@ -768,7 +837,7 @@ XKit.extensions.xcloud = new Object({
 			m_to_add.preferences = m_data;
 			m_to_add.enabled = XKit.installed.enabled(installed[i]);
 
-			if ((m_data.length / 1024 / 1024) >= 1.5) {
+			if (limit_data && (m_data.length / 1024 / 1024) >= 1.5) {
 				console.log("Skipping " + m_to_add.extension + " because length = " + (m_data.length / 1024 / 1024) + " kilobytes");
```
Comment (finagle29) [inline Extensions/xcloud.js]: kilobytes should be megabytes

**Comment by Wolvan** [comment]
Works pretty well, nicely done. Found a (styling) problem on firefox though:
Chrome: 
image
Firefox:
image

This could honestly just be firefox, my whole XKit seems... more block-y on FF. But just wanted to make you aware of that, @ChuckL 

Otherwise this has a LGTM

**Comment by ChuckL** [comment]
@finagle29 Good catch, I decided just to remove the log message.

@Wolvan Done! I forgot to put the third way of making rounded rectangles in the css.

**Comment by Wolvan** [comment]
LGMT, @homu r+

**Pull request merged**